### PR TITLE
Bugfix/30094 clicking a link from the project directory causes error

### PR DESCRIPTION
**Pull request by NullVoxPopuli**
https://developertown.visualstudio.com/SIL%20International%20Scriptoria/_workitems/edit/30094

Temp solution for now. The real solution will be to implement the public project view.

https://developertown.visualstudio.com/SIL%20International%20Scriptoria/_queries/edit/29633/?triage=true

### CODE REVIEW

File: source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/project-directory/show/index.tsx [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+export const pathName = '/directory/:projectId';
+
+export default props => "TBD";
```
Comment (giancorzo) [inline source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/project-directory/show/index.tsx]: is this supposed to be the public view of a project?

File: source/SIL.AppBuilder.Portal.Frontend/src/ui/components/project-table/table/index.tsx [hunk 2]
```diff
@@ -14,6 +14,7 @@ import LoadingWrapper from '@ui/components/loading-wrapper';
 interface IOwnProps {
   projects: ProjectResource[];
   isLoading?: boolean;
+  projectPath?: (id: string) => string;
```
Comment (giancorzo) [inline source/SIL.AppBuilder.Portal.Frontend/src/ui/components/project-table/table/index.tsx]: where is project path coming from?

File: source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/project-directory/show/index.tsx (hunk 1 above)
Comment (NullVoxPopuli) [reply source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/project-directory/show/index.tsx]: Correct.

File: source/SIL.AppBuilder.Portal.Frontend/src/ui/components/project-table/table/index.tsx (hunk 2 above)
Comment (NullVoxPopuli) [reply source/SIL.AppBuilder.Portal.Frontend/src/ui/components/project-table/table/index.tsx]: The directory display

**Pull request merged**